metadata.yaml

```diff
@@ -46,3 +46,7 @@ requires:
         interface: nginx-route
         limit: 1
         optional: true
+
+peers:
```

Comment:
What is the reason for needing this please?

Reply:
My bad, this should be removed.